classes/local/job/memberships_job.php

```diff
@@ -349,6 +349,9 @@ public static function sync_memberships($trace) {
                                 'instanceid' => $enrolmentinstance->id
                             ]
                         );
+                        if(!is_object($membershipsjobpersistent) || $membershipsjobpersistent === false) {
```

Comment:
@vinod4 Two things. One, you need a space between the `if` and `(`. Second your checks here are redundant - that is, `$membershipsjobpersistent` could never be false and get by !is_object (since false isn't an object). That being said, you can really just reduce this down to `if (!$membershipsjobpersistent) {`

Reply:
Hi @ericmerrill  - Thank you. Made the changes in the code.